mPower2/mPower2/View Controllers/TaskBrowserViewController.swift

```diff
@@ -150,11 +150,10 @@ class TaskBrowserViewController: UIViewController, RSDTaskViewControllerDelegate
     
     // MARK: RSDTaskViewControllerDelegate
     open func taskController(_ taskController: RSDTaskController, didFinishWith reason: RSDTaskFinishReason, error: Error?) {
-        // Inform our delegate that we finished a task
-        self.delegate?.taskBrowserDidFinish(task: taskController.taskPath, reason: reason)
-        
         // dismiss the view controller
         (taskController as? UIViewController)?.dismiss(animated: true) {
+            // Inform our delegate that we finished a task
+            self.delegate?.taskBrowserDidFinish(task: taskController.taskPath, reason: reason)
```

Comment:
WARNING: Code stomp. Please revert.

I moved this on purpose. The UI design requires showing the task using a navigation controller push without animation so that the completion task is displayed as the modal is being dismissed.